io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Raw input support
+  - --device=input
```

Comment:
this is NOT supported. It will fail on virtually any distro as there is no stable version of flatpak that has it. Also it's redundant with ALL.

Reply:
hmm, the flatpak documentation mentioned it being available since 1.15.6 (I have 1.15.8 on my system) - is it not correct ? https://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
1.15.x is NOT a stable release. It is not available on most distribution.
It is an incompatible change. 

See https://github.com/flatpak/flatpak/issues/5681 if you don't believe me.